files/image_config/topology/topology.sh

```diff
@@ -38,8 +125,10 @@ stop() {
     TOPOLOGY_SCRIPT="topology.sh"
     PLATFORM=`sonic-cfggen -H -v DEVICE_METADATA.localhost.platform`
     HWSKU=`get_hwsku`
-    if [[ $HWSKU != "" ]]; then
+    if [[ $HWSKU == "msft_"* ]]; then
```

Comment:
why do need to check of specific skus here?

Reply:
This is to keep our existing support of multi-asic pizza box DUTs.. for that, we had introduced msft_* hw_skus..